pages/code-of-conduct.js

```diff
@@ -1,31 +1,31 @@
 import Layout from '../components/layout'
 export default () => (
   <Layout>
-    <div className="measure lh-copy center">
-    <h1>Welcome to DetroitJS</h1>
-    <h2 id="codeofconduct">Code of Conduct</h2>
+    <div className="measure lh-copy center ph3">
+        <h1>Welcome to DetroitJS</h1>
+        <h2 id="codeofconduct">Code of Conduct</h2>
```

Comment:
Can you move the `ph3` up a level to a new containing `div`? This would prevent the padding from affecting the content width.

So something like
```jsx
...
<div className="ph3">
  <div className="measure lh-copy center">
...
```